samples/msal-node-samples/auth-code-distributed-cache/src/AxiosHelper.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import axios from "axios";
+
+class AxiosHelper {
+
+    /**
+     * Makes an Authorization "Bearer" request with the given accessToken to the given endpoint.
+     * @param endpoint
+     * @param accessToken
+     */
+    static async callDownstreamApi(endpoint: string, accessToken?: string, params?: Record<string, string>): Promise<any> {
+        console.log(`Request to ${endpoint} made at: ${new Date().toString()}`);
+
+        const response = await axios.get(endpoint, {
+            headers: (accessToken && { Authorization: `Bearer ${accessToken}` }) || undefined,
+            params: params || undefined
+        });
+
+        if (response.status !== 200) {
+            throw new Error(`Response: ${response.status}`);
+        }
+
+        return (await response.data);
```

Comment:
I'm curious, why is there an await here?

Reply:
habit from the fetch API (where .json() is async). You're right, this is superflous here -removed